client/blocks/login/index.jsx

```diff
@@ -50,6 +52,10 @@ class Login extends Component {
 		window.location.href = this.props.redirectLocation || window.location.origin;
 	};
 
+	setNotice = ( notice ) => {
```

Comment:
Could we reset the notice in `handleValidUsernamePassword` ?
Otherwise it might confuse users who tries again and still see the old error:
![image](https://cloud.githubusercontent.com/assets/230230/26115850/7dd3e0da-3a61-11e7-8c7b-9c119a5cd733.png)